tests/x11/firefox/firefox_passwd.pm

```diff
@@ -74,7 +74,10 @@ sub run {
     $self->firefox_preferences;
     assert_and_click('firefox-passwd-security');
     send_key_until_needlematch([qw(firefox-primary-passwd-selected firefox-passwd-master_setting)], 'alt-shift-u', 4, 2);
-    send_key 'spc' unless check_screen('firefox-passwd-master_setting', 3);
```

Comment:
This is breaking tests
https://openqa.suse.de/tests/18558497#step/firefox_passwd/16

Reply:
That's because the primary-passwd-selected-20250719 needle is lacking click coordinates.
I've updated the needle. Please feel free to restart and verify if it works.

Reply:
Test has to press spc to get to password window